### PR TITLE
minor changes regarding the checkpoints

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8924

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md [hunk 1]
```diff
@@ -575,7 +575,7 @@ Once you select a language and a keyboard layout, your company branded sign-in s
 > [!TIP]
 > If you recieve a message that "Something went wrong" and it "Looks like we can't connect to the URL for your organization's MDM terms of use" then verify you have correctly [assigned licenses](https://docs.microsoft.com/mem/intune/fundamentals/licenses-assign) to the current user.
```
Comment (illfated) [inline windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md:576]: I could be mistaken, but I think this part of the sentence can be adjusted for readability:
```suggestion
> If you recieve a message that "Something went wrong" and it "Looks like we can't connect to the URL for your organization's MDM terms of use", verify that you have correctly [assigned licenses](https://docs.microsoft.com/mem/intune/fundamentals/licenses-assign) to the current user.
```

Comment (illfated) [reply windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md:576]: BTW, this is outside the scope of your changes, I am only taking the opportunity to improve additional content.

File: windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md [hunk 2]
```diff
@@ -573,9 +573,9 @@ Soon after reaching the desktop, the device should show up in Intune as an **ena
 Once you select a language and a keyboard layout, your company branded sign-in screen should appear. Provide your Azure Active Directory credentials and you're all done.
 
 > [!TIP]
-> If you recieve a message that "Something went wrong" and it "Looks like we can't connect to the URL for your organization's MDM terms of use" then verify you have correctly [assigned licenses](https://docs.microsoft.com/mem/intune/fundamentals/licenses-assign) to the current user.
+> If you recieve a message that "Something went wrong" and it "Looks like we can't connect to the URL for your organization's MDM terms of use", verify that you have correctly [assigned licenses](https://docs.microsoft.com/mem/intune/fundamentals/licenses-assign) to the current user.
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md:576]:
```suggestion
> If you receive a message that "Something went wrong" and it "Looks like we can't connect to the URL for your organization's MDM terms of use", verify that you have correctly [assigned licenses](https://docs.microsoft.com/mem/intune/fundamentals/licenses-assign) to the current user.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Pull request merged**